packages/navigation/src/hooks/useWebviewNavigate.ts

```diff
@@ -105,6 +105,9 @@ function getMinimalAction(
   while (
     currentAction.payload &&
     'name' in currentAction.payload &&
+    currentAction.payload.name &&
```

Comment:
Fixes infinite loop as `name` could be `undefined` and `currentState?.routes[currentState.index ?? -1]?.name` could also be undefined.